dev/release/src/release.ts

```diff
@@ -256,7 +286,108 @@ Key dates:
 - It will be deployed to k8s.sgdev.org within approximately one hour (https://k8s.sgdev.org/site-admin/updates)
 - ${releaseBlockerMessage}
             `
-            await postMessage(message, 'dev-announce')
+            await postMessage(message, c.slackAnnounceChannel)
+        },
+    },
+    {
+        id: 'patch:issue',
+        run: async ({ captainGitHubUsername, slackAnnounceChannel }, version) => {
+            const parsedVersion = semver.parse(version, { loose: false })
+            if (!parsedVersion) {
+                throw new Error(`version ${version} is not valid semver`)
+            }
+            if (parsedVersion.prerelease.length > 0) {
+                throw new Error(`version ${version} is pre-release`)
+            }
+
+            // Create issue
+            const { url, created } = await ensurePatchReleaseIssue({
+                version: parsedVersion,
+                assignees: [captainGitHubUsername],
+            })
+            const existsText = created ? '' : ' (already exists)'
+            console.log(`Patch release issue URL${existsText}: ${url}`)
+            if (!created) {
+                return
+            }
+
+            // - Announce issue if issue does not already exist
+            await postMessage(
+                `:captain: Patch release ${parsedVersion.version} will be published soon. If you have changes that should go into this patch release, please add your item to the checklist in the issue description: ${url}`,
+                slackAnnounceChannel
+            )
+            console.log(`Posted to Slack channel ${slackAnnounceChannel}`)
+        },
+    },
+    {
+        id: 'release:publish',
+        run: async ({ slackAnnounceChannel }, version) => {
+            const parsedVersion = semver.parse(version, { loose: false })
+            if (!parsedVersion) {
+                throw new Error(`version ${version} is not valid semver`)
+            }
+            if (parsedVersion.prerelease.length > 0) {
+                throw new Error(`version ${version} is pre-release`)
+            }
+            const requiredCommands = ['comby', 'sed', 'find']
+            for (const cmd of requiredCommands) {
+                try {
+                    await commandExists(cmd)
+                } catch (err) {
+                    throw new Error(`Required command ${cmd} does not exist`)
+                }
+            }
+
+            const changes = [
```

Comment:
It would be nice to type this.

```suggestion
            const changes: octokit.PullsCreateParams & CreateBranchWithChangesOptions  = [
```